src/Support/CompilableFile.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+
+export class CompilableFile {
```

Comment:
Hmm, I'm not sure this name.

Reply:
If we use the name, how about this?

```ts
class CompilableExts {
 exts = ['pb', 'tflite'];
}

// activate() in extension.ts
// ...
vscode.commands.executeCommand('setContext', 'onevscode.compilableExtList', CompilableExts());
```

Reply:
> Hmm, I'm not sure this name.

Could you let me know the reason why you  don't like the name `CompilableFile`?